core/src/main/java/kafka/log/remote/RemoteLogManager.java

```diff
@@ -470,7 +479,8 @@ public void onLeadershipChange(Set<Partition> partitionsBecomeLeader,
      * @param errorHandler   callback to handle any errors while stopping the partitions.
      */
     public void stopPartitions(Set<StopPartition> stopPartitions,
-                               BiConsumer<TopicPartition, Throwable> errorHandler) {
+                               BiConsumer<TopicPartition, Throwable> errorHandler,
+                               String remoteLogDisablePolicy) {
```

Comment:
Why do we assume that the `remoteLogDisablePolicy` is common for all the partitions in the StopPartition request?

Reply:
Sorry, I didn't get your question here. I added the 3rd parameter here because most of the logic is similar, just a few differences. Do you have any suggestion here?

Reply:
I see. Will update it later.

Reply:
Added `remoteLogDisablePolicy` into `StopPartition` as new class field.